### DISCUSSION TITLE
cmark-gfm et CVE-2022-39209.

### DISCUSSION THREAD

**Discussion by Passionlinux** [KDE/ghostwriter#828] [Q&A]
Hello, I am the maintainer for Debian (as well as one of the maintainers for openSUSE), I would have a request because for the moment the package is awaiting correction at Debian.

Unfortunately, my package was refused for various easily correctable reasons (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1021364), but above all for a security problem (CVE-2022-39209 ).

Can you put the cmark-gfm library in a secure version?

Thanks in advance.

**Comment by wereturtle** [comment]
Done.  :)

**Reply by Passionlinux** [reply]
thank you very much, I'm tweaking the package as requested, then I'll come back to you if you want to incorporate all this at source (if you want).

**Comment by Passionlinux** [comment]
Sorry to bother you again, can you make a minor release to mark the change?

Thanks in advance.
